### ISSUE TITLE
Make cx16assem run as an external command inside cx16shell

### ISSUE THREAD

**Issue by mgkaiser** [irmen/cx16assem#1]

**Comment by irmen** [comment]
Can you elaborate?
Because you can already just launch it when you type the name.

**Comment by mgkaiser** [comment]
Right, but it drops back to BASIC before doing so rather than running subordinate to the shell.  I know maybe you meant the shell solely as a “curiosity” but I think it is a contender to be the “primary” CLI for the Cx16 if stuff supports it.

**Comment by irmen** [comment]
I see. Well, the assembler is developed as a stand alone program. 
And the shell is as well.
The X16 is not a platform to multitask / process switch easily, so this remains an impossibility right now.

**Comment by mgkaiser** [comment]
Not looking for multitasking.  Just looking for a way to quickly move among the editor, assembler, run the program it built.  Think “CP/M”, not “Linux”.   The Cx16 is more capable than most CP/M machines ever were.  The SDCard is faster than any floppy back in the day was and CP/M rarely had more than 64k available and maxed out at 128k in 3.0

**Comment by irmen** [comment]
I agree that the specs "could" allow it, but for now I think you have to stick with the edit/assemble/run cycle as it is supported in the command mode of the assembler itself...

**Comment by mgkaiser** [comment]
I’m gonna give it a try.  Remember I had not gotten the assembler to run previously.  So far I like it.
